tools/ci-ts/test-helpers/chainlinkClient.test.ts

```diff
@@ -0,0 +1,150 @@
+let containerMockState = {
+  Paused: false,
+}
+
+function setContainerMockState(newState: any) {
+  containerMockState = newState
+}
+
+const containerMock = {
```

Comment:
If having all the mocks within this file seems a bit unwieldy, they can be moved to their own `__mocks__` folder which jest will load when matched with a `jest.mock` AFAIK